src/csharp/SDK/Native/NativeMethods.cs

```diff
@@ -217,7 +226,7 @@ k4a_memory_destroy_cb_t free
             out Vector2 target_point2d,
```

Comment:
You missed the k4a_calibration_2d_to_2d calling convention for you Cdecl => k4aCallingConvetion.